sphinx/conf.py

```diff
@@ -20,6 +22,8 @@
 sys.path.append(os.path.abspath("./"))
 #: The path to the django settings module (see :doc:`sphinxcontrib-django2:readme`)
 django_settings = "backend.sphinx_settings"
+# Split by "." and take the first two parts to get "major.minor"
+django_minor_version = ".".join(django_version.split(".")[:2])
```

Comment:
Totally okay, but did you know that there is also another solution:
```
>>> import django
>>> django.VERSION
(2, 0, 0, 'final', 0)
```

Reply:
I did, but at the time I couldn't come up with a better solution than `".".join(map(str, django.VERSION[:2]))` (which isn't that much better)... after giving it some thought, maybe `f"{django.VERSION[0]}.{django.VERSION[1]}"` is indeed more readable?

Reply:
I updated the PR accordingly.